perspective.el

```diff
@@ -208,10 +211,11 @@ Run with the activated perspective active.")
 
 (defun perspectives-hash (&optional frame)
   "Return a hash containing all perspectives in FRAME.
-FRAME defaults to the currently selected frame. The keys are the
-perspectives' names. The values are persp structs, with the
-fields NAME, WINDOW-CONFIGURATION, BUFFERS, BUFFER-HISTORY,
-KILLED, POINT-MARKER, and LOCAL-VARIABLES.
+FRAME defaults to the currently selected frame.
+The keys are the perspectives' names.
+The values are persp structs, with the fields NAME,
+WINDOW-CONFIGURATION, BUFFERS, BUFFER-HISTORY, KILLED,
+POINT-MARKER, and LOCAL-VARIABLES.
```

Comment:
Are you just re-wrapping to a narrower column? If so, why? To avoid double-spaces after periods? (which, I disagree with).

Reply:
add to custom: `(sentence-end-double-space nil)`